media_driver/linux/common/vp/ddi/media_libva_vp.c

```diff
@@ -4179,6 +4185,9 @@ DdiVp_QueryVideoProcFilterCaps (
             {
                 VAProcFilterCapTotalColorCorrection* TccCap;
 
+                if (uQueryCapsNum < uExistCapsNum)
+                    return VA_STATUS_ERROR_MAX_NUM_EXCEEDED;
```

Comment:
Here, to make the coding style consistent, is it possible to use {}? Thanks a lot in advance!

Reply:
Thanks for the comment, I updated the patch.